static/app/views/performance/trends/types.tsx

```diff
@@ -56,6 +56,18 @@ export enum TrendColumnField {
   SPANS_RESOURCE = 'spans.resource',
 }
 
+export enum TrendParameterField {
```

Comment:
Let's change this to `TrendParameterLabel` and the other from `TrendColumnField` to `TrendParameterColumn`.